packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1170,6 +1180,31 @@ describe('network stubbing', { retries: 2 }, function () {
           })
         })
       })
+
+      context('with `times`', function () {
+        it('only uses each handler N times', function () {
+          const third = sinon.stub()
+
+          cy.intercept({ url: '/foo', times: 1 }, 'first')
+          .intercept({ url: '/foo', times: 2 }, 'second')
+          .intercept({ url: '/foo', times: 2 }, third)
+          .intercept({ url: '/foo', times: 3 }, 'fourth')
+          .then(async () => {
+            const expectGet = (expected) => $.get('/foo').then((res) => expect(res).to.eq(expected))
+
+            await Promise.mapSeries([
+              'first',
+              'second',
+              'second',
+              'fourth',
+              'fourth',
+              'fourth',
+            ], expectGet)
+
+            expect(third).to.be.calledTwice
+          })
+        })
+      })
```

Comment:
@brian-mann thoughts on this? I know you weren't originally the biggest fan of this API, but after some research, I think it's still the most direct way to accomplish the user ask